packages/components/src/components/primitives/PinInput/PinInput.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { FC } from 'react';
+import {
+  Flex,
```

Comment:
I don't think we need that but in case we do, we should try to import it from our `@components` library @TheEnmanuel23

Reply:
I modified the import to use it from `@components` I didn't know we have it there, thanks!